misc/manifest.xml

```diff
@@ -2,7 +2,7 @@
 <ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
   <ProviderNameSpace>Microsoft.GuestConfiguration</ProviderNameSpace>
```

Comment:
i am not able to add comment on gc_1.0.0.zip package so adding it here.
seems like this pr is modifying the package. what changes the new zip package has?
yesterday i updated the binaries with latest code.... you dont need to update it ...

Reply:
@amitsara I am finally able to get back to this. This PR isn't updating the .zip. It is only pushing it to an artifact feed automatically when extension builds succeed.

Reply:
So it can be consumed by the Module build script